pKa_challenge_instructions.md

```diff
@@ -130,6 +130,15 @@ Newly added microstates will also be shared with participants and microstates li
 Please do not create a microstate ID yourself. 
 It is important that challenge organizers assign unique microstate IDs and keep track.  
 
+### Updates on microstates lists
+
+Microstate SMILES strings and microstate IDs can be found in `physical_properties/pKa/microstates/` directory. 
+Due to replicate and missing microstates present in the first release of microstate lists, we have updated `SMXX_microstates.csv` files with necessary corrections in Version 1.4 of this repository. 
+The main correction of this update was the removal of resonance structures that were causing dublicate representation of same microstates. 
+We have also added new microstates suggested by participants.
+Newly added microstates were assigned unique microstate IDs, as recorded in `SMXX_microstates.csv` files.
```

Comment:
Same thing I said about the directory paths, it would be convenient for the file names to also link to the relevant file.

Reply:
These files are all in the directory mentioned at the beginning of paragraph. I added a link to the directory. I won't add link to the files.

Reply:
I think since it's multiple files you can't have each be a link.

Reply:
Correct.